pysat/instruments/ucar_tiegcm.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't think this belongs here...

Reply:
No, it shouldn't be in the pull. I didn't quite clear out my last changes before making this pull request. I'm fine with dealing with the TIEGCM pull first.